napari/utils/settings/_defaults.py

```diff
@@ -66,6 +66,9 @@ class ApplicationSettings(BaseSettings, EventedModel):
     first_time: bool = True
 
     # Window state, geometry and position
+    save_window_geometry: bool = Field(
+        False, description="Save window size and position."
```

Comment:
I can't remember now, but I thought @jni had said last week he was fine remembering window geometry by defualt, he just wanted to be able to turn it off if he wanted to?

Personally I've really enjoyed having it be remembered, and I know that was a time when we were thinking we wouldn't even give users control over this as it is so helpful?

Maybe @jni can weigh in before merge?

Reply:
@goanpeca also mentioned to me that it should probably be set to True as the default, so I went ahead and made this change.

Reply:
Great, I'm all for merging this now then as is, and we can decide if we maybe want to push button instead later - but curious what @jni thinks before final decision